dispute_evidence.go

```diff
@@ -0,0 +1,12 @@
+package braintree
+
+import "time"
+
+type DisputeEvidence struct {
+	XMLName           string     `xml:"evidence"`
+	Comment           string     `xml:"comment"`
+	CreatedAt         *time.Time `xml:"created-at"`
+	ID                string     `xml:"id"`
+	SentToProcessorAt string     `xml:"sent-to-processor-at"`
```

Comment:
I want to make this field and the other date fields being added an object, maybe `*time.Time`, but it's not straight forward because the object is only a date and not a full date time. 🤔